BUILD.md

```diff
@@ -1,105 +1,131 @@
 WDT opensource build uses CMake 3.2.x or later
 
 This should build in a variety of environments (if it doesn't please open
-and issue or even better, contribute a patch)
+an issue or even better, contribute a patch)
 
 See also README.md Dependencies section
 (Inside facebook see https://our.intern.facebook.com/intern/wiki/WDT/OpenSource)
 
-Notes:
- On Ubuntu 14.04 - to get g++ 4.9
+# Notes:
+On Ubuntu 14.04 - to get g++ 4.9
+ ```
  sudo add-apt-repository ppa:ubuntu-toolchain-r/test
  sudo apt-get upgrade
+ ```
 If using a vmware image/starting fresh
+```
  sudo vmware-config-tools.pl  # to setup shared folders
-
-# Cmake 3.2 or later
-
-See below for mac (Cmake 3.3/head on a mac is recommended for Xcode support)
-
+```
+# Build Instructions
+__Install Cmake 3.2 or greater.__
+*See directions below for Mac (Cmake 3.3/head on a Mac is recommended for Xcode support)*
+```
 wget http://www.cmake.org/files/v3.2/cmake-3.2.3.tar.gz
 tar xvfz cmake-3.2.3.tar.gz
 cd cmake-3.2.3
 ./bootstrap --prefix=/usr --parallel=16 && make -j && sudo make install
-
-# Get folly
+```
+__Install folly.__
```

Comment:
Actually this doesn't install folly - just fetches the source tree - can you change to "__Get folly's source tree__" ?